src/sdk.ts

```diff
@@ -255,9 +268,11 @@ const _getSessionData = (
 
 const _getSessionAccounts = async (
     sessionKey: string,
+    privateKey: string,
```

Comment:
Can we be consistent and expect a `NodeRSA.Key` like the other functions in the SDK?